package.json

```diff
@@ -193,7 +193,6 @@
     "react-dom": "^16.9.0",
     "react-dropzone": "^11.3.1",
     "react-flow-renderer": "^10.3.16",
-    "react-google-charts": "^2.0.0",
```

Comment:
i simply ran `yarn remove react-google-charts`, I hope its good the way it is now :)